content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Introducing the Istio Operator
+description: Introduction to Istio's new operator-based installation and control plane management feature.
+publishdate: 2019-11-12
+subtitle:
+attribution: Martin Ostrowski (Google), Frank Budinsky (IBM)
+keywords: [install,configuration,istioctl,operator]
+target_release: 1.4
+---
+
+Kubernetes [operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) provide
+a pattern for encoding human operations knowledge in software and are a popular way to simplify
+the administration of software infrastructure components.
+Because Istio is complex to administer, it's a natural candidate for an operator.
+
+Up until now, [Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio.
+This approach worked well but had some disadvantages:
+
+1. Users need to install another tool.
+1. Supporting Istio's many small specializations cause Helm templates to become unwieldy.
+1. Helm configurations are difficult to validate using Istio's proto-based APIs.
+1. Upgrades sometimes require Istio specific hooks that have been difficult to implement and maintain with Helm.
+
+Starting with Istio 1.4, the [Helm installation](/docs/setup/install/helm/) approach has been deprecated
```

Comment:
What does deprecated mean? How long will we continue to support Helm?

Migrating from Helm to the operator is going to be a huge change for existing users. We should assure them exactly how long they have to do this migration.

@ostromart mentioned we are not recommending existing users migrate from helm to the operator. Is this the case? Should we express that here?

Reply:
Since [Helm is a Beta feature](https://istio.io/about/feature-stages/#core), the standard 3 months [deprecation policy](https://istio.io/about/feature-stages/) applies. I think that goes without saying, but we could mention it in the release notes.

Reply:
That would imply we would delete Helm in 1.5 which I think is not the plan, so we should probably call it out here?

Reply:
I kind of agree with John. I think we should use the word "recommended" instead of "deprecated" (e.g. Starting with Istio 1.4, a new installation using istioctl is recommended, instead of the Helm installation approach.) Deprecated has a **very specific** documentation meaning that seems likely to be interpreted the way John suggests. We should only use it if we intend to actually follow its meaning.

Reply:
Yes, let's not use the word deprecate externally, we'll not be deleting Helm in 3 months. We want to encourage users to switch but migration tools are still alpha. I think we could say "plan to deprecate" or just "recommend istioctl" is probably better for now.